packages/flutter/test/material/progress_indicator_test.dart

```diff
@@ -255,4 +255,47 @@ void main() {
     await tester.pump(const Duration(milliseconds: 1));
     expect(tester.hasRunningAnimations, isTrue);
   });
+
+  testWidgets('LinearProgressIndicator with height 12.0', (WidgetTester tester) async {
```

Comment:
The developer was concerned about creating a linear progress indicator with a height that was less than the default (too), we should verify that works.